### ISSUE TITLE
runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by Y-Less** [Southclaws/sampctl#108]
```
runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x20 pc=0x9a92c7]

goroutine 23 [running]:

github.com/Southclaws/sampctl/runtime.run.func5
(0xc0430b1e20, 0xc04208e2c0, 0xc042048c60)
        /home/travis/gopath/src/github.com/Southclaws/sampctl/runtime/run.go:155 +0x1b7
created by github.com/Southclaws/sampctl/runtime.run
        /home/travis/gopath/src/github.com/Southclaws/sampctl/runtime/run.go:141 +0x40d
```

Probably the same as #95 but that didn't really focus on the panic, only the display.  This one too was mixed in with other text, but I cleaned it up.

This is 1.6.2, since I've not updated to 1.6.3 yet due to #107 .

**Comment by Southclaws** [comment]
I fixed the 1.6.3 windows release now so try again. I did fix a couple of panics in 1.6.3 so it might have already been solved.

**Comment by Southclaws** [comment]
Yes, I checked and I have fixed that one, it was an attempted deref on the error pointer returned from the command runner.
